scripts/legacy/scil_visualize_bundles_mosaic.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from scilpy.io.deprecator import deprecate_script
+from scripts.scil_bundle_screenshot_mosaic import main as new_main
+
+
+DEPRECATION_MSG = '''
+This script has been renamed scil_bundle_screenshot_mosaic.py. Please change
```

Comment:
FYI I tested with the test that is commented in test_visualize_bundles_mosaic: on both master and your branch, result is the same ugly thing, can be managed later.
![test_visualize_bundles_mosaic_out_master](https://github.com/scilus/scilpy/assets/4967417/04952d96-a8e2-4a1e-a36d-6a2c8ea50f5f)

Reply:
Can you give ma data and a command line to test this ? It should not be hard at all to fix the font sizing in this PR. I do something similar for slice labelling in the screenshot scripts